darwin/future/core/client.py

```diff
@@ -155,7 +155,7 @@ def _setup_session(self, retries: Retry) -> None:
 
     @property
     def headers(self) -> Dict[str, str]:
-        http_headers: Dict[str, str] = {"Content-Type": "application/json"}
+        http_headers: Dict[str, str] = {"Content-Type": "application/json", "Accept": "application/json"}
```

Comment:
This was required for the stage api, but if we need to set this for other api's we'll need to think about a syntax for setting headers of the client, either passing headers as an optional parameter to the client.get(...)/REST functions or changing headers before certain API calls